### DISCUSSION TITLE
AniMe Matrix Text support

### DISCUSSION THREAD

**Discussion by TechNo1geek** [seerge/g-helper#1253] [Ideas]
I love the app, however i seem to be ging back to armour crate for the text on AniMe matrix

**Comment by seerge** [comment]
@TechNo1geek i eventually plan to extend anime matrix editor a bit including some text editing there
https://github.com/seerge/g-helper/issues/706

**Reply by BlueForrest** [reply]
Within AC we have 2 text lines on the Anime editor, in G-Helper this does not exist at this moment.
Any Idea when free text lines on Anime would be possible?

**Reply by seerge** [reply]
You can just make a GIF with a text in it and set it that way as well :)

**Reply by BlueForrest** [reply]
Great, did not thought in that direction. Thanks
